i18n/en-US.yml

```diff
@@ -28,16 +28,45 @@ _name: English
 components:
   DefaultItinerary:
     clickDetails: Click to view details
-    # Use ordered placeholders for the departure-arrival string
-    # (this will accommodate right-to-left languages by swapping the order in this string).
-    departureArrivalTimes: "{startTime}—{endTime}"
     # Use ordered placeholders when multiple modes are involved
     # (this will accommodate right-to-left languages by swapping the order/separator in this string).
     multiModeSummary: "{accessMode} to {transitMode}"
-    # If trip is less than one hour only display the minutes.
-    tripDurationFormatZeroHours: "{minutes, number} min"
-    # TODO: Distinguish between one hour (singular) and 2 hours or more?
-    tripDurationFormat: "{hours, number} hr {minutes, number} min"
+  ItinerarySummary:
+    calories: Cal
+    fareCost: "{useMaxFare, select {
+      true {{minTotalFare} - {maxTotalFare}}
+      false {{minTotalFare}}
+    }}"
+    transfers: "{transfers} {transfers, plural, =0 {Transfer} other {Transfers}"
```

Comment:
There is an issue with this expression: 
![image](https://user-images.githubusercontent.com/56846598/126352530-bc96e7d9-b6a8-4c18-a6c8-67f1cab5226d.png)

In `example.js`, set: 
```js
  ItineraryBody: LineItinerary, // (add import)
```
and make an itinerary search in mobile view.

Reply:
This should be addressed in the latest commit (#1fbee66) 

![image](https://user-images.githubusercontent.com/63798641/126394618-bf68f63c-584c-4944-9d18-eb0a63ff4497.png)